cpp/ql/src/semmle/code/cpp/controlflow/internal/ConstantExprs.qll

```diff
@@ -104,9 +104,42 @@ private predicate loopConditionAlwaysUponEntry(ControlFlowNode loop, Expr condit
   )
 }
 
+/**
+ * This relation is the same as the `el instanceof Function`, only obfuscated
+ * so the optimizer will not understand that any `FunctionCall.getTarget()`
+ * should be in this relation.
+ */
+pragma[noinline]
+private predicate isFunction(Element el) {
+  el instanceof Function
+  or
+  el.(Expr).getParent() = el
+}
+
+/**
+ * Holds if `fc` is a `FunctionCall` with no return value for `getTarget`. This
+ * can happen due to extractor issue CPP-383.
```

Comment:
Can we link to the new issue from Luke here?

Reply:
The problem of no `getTarget` value can in principle be caused by any extractor crash, so I've changed this comment to avoid mentioning a specific issue.